python/.gitignore

```diff
@@ -0,0 +1,161 @@
+.venv
```

Comment:
I guess we should put all the node stuff in a .gitignore local to the js and typescript dirs, eh?